### PR TITLE
fix(#676):style tweaks to match UI design specs

### PR DESCRIPTION
**Pull request by chrisolsen**

### CODE REVIEW

File: libs/docs/src/components/common/Button.stories.mdx [hunk 1]
```diff
@@ -46,7 +48,6 @@ There are three main button types: Primary, Secondary, and Tertiary. If there is
         <goa-button type="primary">Primary</goa-button>
         <goa-button type="secondary">Secondary</goa-button>
         <goa-button type="tertiary">Tertiary</goa-button>
-        <goa-button type="get-started">Get Started</goa-button>
       `}
```
Comment (bluekurinji) [inline libs/docs/src/components/common/Button.stories.mdx:51]: start type button is displayed, but does not have code snippet for it

File: libs/docs/src/components/common/Button.stories.mdx [hunk 2]
```diff
@@ -178,6 +176,61 @@ There are two button sizes: Normal (default) and Compact.
...
+    <GoAButton type="primary" trailingIcon="airplane" onClick={noop}>Trailing</GoAButton>
+    <GoAButton type="primary" leadingIcon="archive" trailingIcon="airplane" onClick={noop}>Both</GoAButton>
+  </GoAButtonGroup>
+</Story>
+
+<Tabs>
+  <Tab label="Web Component">
+    <CodeSnippet
+      code={`
+        <goa-button type="primary">Primary</goa-button>
```
Comment (bluekurinji) [inline libs/docs/src/components/common/Button.stories.mdx:194]: The code snippet does not have icons related reference in it

File: libs/docs/src/components/common/Button.stories.mdx [hunk 3]
```diff
@@ -160,6 +157,7 @@ There are two button sizes: Normal (default) and Compact.
   </GoAButtonGroup>
 </Story>
 
+
 <Note type="do">
```
Comment (bluekurinji) [inline libs/docs/src/components/common/Button.stories.mdx:161]: The do and don't may have been there earlier. As per the current specifications, story books are not expected to have them. Please consider removing it

File: libs/react-components/src/lib/button-group/button-group.css [hunk 4]
```diff
@@ -3,9 +3,8 @@ goa-button-group > goa-button ~ goa-button {
   margin-top: 0.25rem;
 }
 
-@media (min-width: 320px) {
+@media (min-width: 480px) {
```
Comment (bluekurinji) [inline libs/react-components/src/lib/button-group/button-group.css:6]: As per the recent decision suggestion from you, the px need to be in rem???

File: libs/web-components/src/components/button/Button.svelte [hunk 5]
```diff
@@ -57,7 +57,7 @@
     box-sizing: border-box;
     font-family: var(--font-family);
   }
-  @media (max-width: 320px) {
+  @media (max-width: 480px) {
```
Comment (bluekurinji) [inline libs/web-components/src/components/button/Button.svelte:60]: As per the recent decision suggestion from you, the px need to be in rem???

File: libs/web-components/src/components/button/Button.svelte [hunk 6]
```diff
@@ -68,27 +68,28 @@
...
+    box-sizing: border-box;
     border-radius: 0.25rem;
     border: 2px solid var(--goa-color-interactive);
     box-sizing: border-box;
     cursor: pointer;
+    font-family: var(--font-family);
     font-size: var(--fs-lg);
     font-weight: 400;
     height: var(--button-height);
+    letter-spacing: 0.5px;
```
Comment (bluekurinji) [inline libs/web-components/src/components/button/Button.svelte:80]: does it have to be px or rem?

**Review by bluekurinji — COMMENTED** [review]
Also, type="start" is missing in both svelte and react tests

File: libs/web-components/src/components/button/Button.svelte (hunk 5 above)
Comment (chrisolsen) [reply libs/web-components/src/components/button/Button.svelte:60]: you use px for resolution

File: libs/web-components/src/components/button/Button.svelte (hunk 6 above)
Comment (chrisolsen) [reply libs/web-components/src/components/button/Button.svelte:80]: px

File: libs/react-components/src/lib/button-group/button-group.css (hunk 4 above)
Comment (chrisolsen) [reply libs/react-components/src/lib/button-group/button-group.css:6]: px

File: libs/docs/src/components/common/Button.stories.mdx (hunk 3 above)
Comment (chrisolsen) [reply libs/docs/src/components/common/Button.stories.mdx:161]: they have always been there....

File: libs/react-components/src/lib/button-group/button-group.css (hunk 4 above)
Comment (bluekurinji) [reply libs/react-components/src/lib/button-group/button-group.css:6]: We have to come up with some document and stick to it. I remember rem usage recommended for  media query earlier. That need not hold this pull request

File: libs/web-components/src/components/button/Button.svelte (hunk 5 above)
Comment (bluekurinji) [reply libs/web-components/src/components/button/Button.svelte:87]: same as above

**Comment by bluekurinji** [comment]
The size section dose not have type="start". I will leave it to review with Mark.

**Review by bluekurinji — APPROVED** [review]

**Pull request merged**